client/app/domain/domain.service.js

```diff
@@ -1177,6 +1177,40 @@ angular.module('services').service(
       });
     }
 
+    // --------------------- Options ----------------------------
+
+    /**
+     * Get all options
+     * @param {string} serviceName
+     */
+    getOptions(serviceName) {
+      return this.OvhHttp.get(`/domain/${serviceName}/option`, {
```

Comment:
Imo, those should be made through `ovh-api-services` to avoid using multiple data providers

Reply:
All API calls in domain are done this way. So, I thought we can take this task of routing all the calls via ovh-api-services, as a separate story. Please let me know your thoughts :)

Reply:
This doesn't mean that we've got to continue that way. As we are adding new calls, it would be better starting uniformizing now, it's not much but it is also less to do later :)

Reply:
This is done :)